bio.css

```diff
@@ -0,0 +1,71 @@
+* {
+  font-family: Impact, fantasy;
+  font-size: 20px;
+}
+
+body {
+  background-color: lightgray;
+}
+
+.header-nav {
+  display: flex;
+  background-color: darkblue;
+  position: sticky;
```

Comment:
noticed that the navbars on the Home and Bio pages were not sticky on the deployed site, but the navbars on the Experience and Portfolio pages were.